connector/mqtt/latency-pub-test-prometheus/app/LatencyStore.js

```diff
@@ -0,0 +1,111 @@
+const ss = require('simple-statistics');
+const { logger } = require('@dojot/dojot-module-logger');
+
+const TAG = { filename: 'LatencyStore' };
+
+/** Class to store Latencys and calculate statistics * */
+class LatencyStore {
+  /**
+     *
+     */
+  constructor() {
+    logger.debug('Init LatencyStore...', TAG);
+
+    this.latencies = [];
+    this.max = null;
+    this.min = null;
+  }
+
+  /**
+     * Retrieve array of latencies
+     *
+     * @return {Array} All latencies
+     */
+  getLatencies() {
+    return this.latencies;
+  }
+
+  /**
+     * Clean all store info
+     */
+  cleanLatencies() {
+    logger.debug(`Clean all Latencies, the Latencies before clean: ${this.getLatencies()}`, TAG);
+
+    this.latencies = [];
+    this.max = null;
+    this.min = null;
```

Comment:
max and min are string or number?
they should not be initialized with 0?

Reply:
Because I would like to expose the metrics only when I receive a number and zero is a number.